Modules/DataModel/Sources/PocketCastsDataModel/Private/Managers/EndOfYearDataManager.swift

```diff
@@ -205,7 +205,7 @@ class EndOfYearDataManager {
                             WHERE `\(DataManager.podcastTableName)`.uuid = `\(DataManager.episodeTableName)`.podcastUuid and
                                 \(listenedEpisodesThisYear)
                             GROUP BY podcastUuid
-                            ORDER BY played_episodes DESC
+                            ORDER BY totalPlayedTime DESC
```

Comment:
I changed that because if a person listened to, let's say, 40 episodes of a podcast that has 1-minute episodes, that would be their top podcast. Which is weird.